src/crime_vancouver_eda.md

```diff
@@ -0,0 +1,55 @@
+# Exploratory data analysis of Crimes in Vancouver data set
+
+# Summary of the data set
+
+The data set used in this project comes from the Vancouver Police Department Open Data (The Vancouver Police Department, 2021) and can be found here (https://geodash.vpd.ca/opendata/). Each row in the data set represents a reported crime in Vancouver, including crime type, date (in year, month, day, hour, minute), block address, and neighbourhood name. The data set covers cases from 2003 to 2021 of a total of 782,749 cases. For the visualization purpose, we focus on the past ten years' data here (from 2020 to 2021).
+Below we show the number of reported crimes for each neighbourhoods. Note there are 24 neighbourhoods in Vancouver.
+
+![](figure-gfm/neighbour_crimes.png)<!-- -->
+
+Table 1: Count of crimes of each Neighbourhood.
+
+# Data split into training and test sets
+
+Data is split in a way that 75% observations are in the training data set and 25% are for testing. Below we list the counts of observations of both classes:
+
```

Comment:
1. Did you use my src/split_data.py as the EDA? Since I have hard coded it as 80/20 split, if you want to use 75/25 then I will need to change the script.
2. Please make an issue of your data choice so everyone is aware of it in case we switch

Reply:
It's ok I can change mine, this is easier. Let's do 80/20